### PR TITLE
Update species database and benchmark categories for TransportTracer changes in GEOS-Chem 14.2.0

### PR DESCRIPTION
**Pull request by msulprizio**
### Name and Institution (Required)

Name: Melissa Sulprizio
Institution: Harvard / GCST

### Describe the update

In GEOS-Chem 14.2.0, several TransportTracer species were renamed for consistency with GMAO's tracer gridded component (TR_GridComp). Here the species names are fixed and the `PassiveTracers` benchmark category has been renamed to `TransportTracers`.

### Related Github Issue(s)

- https://github.com/geoschem/geos-chem/pull/1816
- https://github.com/geoschem/HEMCO/pull/214

### CODE REVIEW

File: gcpy/species_database.yml [hunk 1]
```diff
@@ -136,6 +136,55 @@ ALK4:
...
+  Is_Gas: true
+  Is_Tracer: true
+  Snk_Horiz: all
+  Snk_Mode: constant
+  Snk_Value: 0
+  Snk_Vert: surface
+  Src_Add: true
+  Src_Horiz: all
+  Src_Mode: constant
+  Src_Units: timestep
```
Comment (yantosca) [inline gcpy/species_database.yml:151]: For units, see comment by @lizziel at https://github.com/geoschem/geos-chem/pull/1816#discussion_r1212282154

Comment (yantosca) [reply gcpy/species_database.yml:151]: Also see link to @lizziel's comment at https://github.com/geoschem/geos-chem/pull/1816#discussion_r1212246849

**Review by yantosca — CHANGES_REQUESTED** [review]
These updates look good but I've noted some comments about the `species_database.yml` file from the geos-chem repo that should be addressed before merging.

File: gcpy/species_database.yml (hunk 1 above)
Comment (msulprizio) [reply gcpy/species_database.yml:148]: The species_database.yml file has now been updated in both the geos-chem repository and here

**Pull request merged**